ci/azure-pipelines-nightly.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Azure Release Pipeline configuration
+# Documentation at https://aka.ms/yaml
+
+schedules:
+# For testing purposes, run this pipeline every day at 07:21 UTC
+# Use a random minute value to avoid congestion on the hour
+- cron: "21 7 * * *"
```

Comment:
nit: 21 is not random. If this was cron I'd do something like `sleep $(($RANDOM % FOO))` you may want to look into azure docs if this is possible.